packages/holocron/src/ducks/compose.js

```diff
@@ -53,6 +53,7 @@ export function composeModules(moduleConfigs) {
           return Promise.resolve();
         })
         .catch((error) => {
+          console.error(`Error while attempting to call 'load' or 'loadModuleData' inside composeModules for ${name}.`, error);
```

Comment:
```suggestion
          console.error(`Error while attempting to call 'loadModuleData' inside composeModules for ${name}.`, error);
```
`load` is deprecated, and I dont see much use of it. We can keep this log a bit shorter by leaving that out

Reply:
Could that be held until the calls to `load` are removed? The error could come from `load` since we're calling it.

Reply:
For further consideration I only modified existing language already in the package:
https://github.com/americanexpress/holocron/blob/main/packages/holocron/src/holocronModule.jsx#L83

IMO it should stay and be removed uniformly when the call is removed.

Reply:
Yah, consistency with the other errors are good